src/codegen/flags.rs

```diff
@@ -58,9 +58,13 @@ pub fn generate(env: &Env, root_path: &Path, mod_rs: &mut Vec<String>) {
         try!(general::uses(w, env, &imports));
         try!(writeln!(w));
 
+        let mut first = true;
         for config in &configs {
             if let Type::Bitfield(ref flags) = *env.library.type_(config.type_id.unwrap()) {
-                mod_rs.push("\nmod flags;".into());
+                if first {
+                    mod_rs.push("\nmod flags;".into());
```

Comment:
Just move it before for

Reply:
Ah true, now the check is before... Good catch!